README.MD

```diff
@@ -45,41 +45,91 @@ In your main style file (e.g. _style.less_) :
 ## Consuming the help component
 
 
-From your Angular `AppModule`:
+### Create a configuration service
+
+This module needs a basic confuration. 
```

Comment:
fixed in https://github.com/tech-advantage/edc-popover-ng/pull/24/commits/5be0fd85afe9379cca243aab7ebae9628ca72b3d